x-pack/plugins/triggers_actions_ui/public/index.ts

```diff
@@ -45,6 +45,7 @@ export function plugin() {
 
```

Comment:
Exporting from `alert_api` is increasing the page load bundle size for `triggersActionsUi` to 90Kb, which is very close to the desired limit of 100Kb.

Can we instead export like this:

```
export { loadAlerts } from './application/lib/alert_api/rules';
export { loadAlertAggregations } from './application/lib/alert_api/aggregate';
```

This will only export the functions that you need

Reply:
Sure, that's even better! I may even export `loadAlertAggregations` alone for now.